From 87489c1f2be3f2e5d8d6859e79c111f2d736e5ed Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Sun, 8 Mar 2026 07:38:18 +0000
Subject: [PATCH 4/4] Do not disable adb authentication in GSI system_ext props

The upstream GSI default of ro.adb.secure=0 leaves ADB completely
unauthenticated on every boot.  Remove the override so the device
tree and boot scripts can control authentication on a per-boot basis.
---
 target/board/gsi_system_ext.prop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/board/gsi_system_ext.prop b/target/board/gsi_system_ext.prop
index 780aadc..223ef97 100644
--- a/target/board/gsi_system_ext.prop
+++ b/target/board/gsi_system_ext.prop
@@ -1,8 +1,8 @@
 # GSI always generate dex pre-opt in system image
 ro.cp_system_other_odex=0
 
-# GSI always disables adb authentication
-ro.adb.secure=0
+# GSI: do not override adb authentication -- let the device tree control it.
+# ro.adb.secure=0
 
 # GSI disables non-AOSP nnapi extensions on product partition
 ro.nnapi.extensions.deny_on_product=true
-- 
2.53.0

